website/views/user.py

```
@@ -1667,3 +1762,544 @@ def delete_notification(request, notification_id):
+        form = RecommendationForm(request.POST)
+        # Pass skills_by_category to template for checkbox rendering
+        try:
+            skills_queryset = RecommendationSkill.objects.all().order_by("category", "name")
+            skills_by_category = {}
+            for skill in skills_queryset:
+                category = skill.category or "Other"
+                if category not in skills_by_category:
+                    skills_by_category[category] = []
+                skills_by_category[category].append((skill.name, skill.name))
+            form.skills_by_category = skills_by_category
+        except Exception:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)